graphql/custom_scalars.go

```diff
@@ -0,0 +1,38 @@
+package graphql
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/99designs/gqlgen/graphql"
+)
+
+// MarshalStringMap handles marshaling StringMap
+func MarshalStringMap(val map[string]string) graphql.Marshaler {
+	return graphql.WriterFunc(func(w io.Writer) {
+		err := json.NewEncoder(w).Encode(val)
+		if err != nil {
+			panic(err)
```

Comment:
Instead of panicking, I would write some sort of error to the writer

Reply:
the Write function returns an error. should I use grip instead?